mobile-app/src/domain/alert/screens/alert.screen.tsx

```diff
@@ -29,33 +29,57 @@ const AlertScreen = () => {
 		location: {} as DeviceLocation,
 	});
 
+	const [previousAppState, setPreviousAppState] =
+		useState<AppStateStatus>('unknown');
+
 	const { fullName } = useUserInfoStore();
 	const { contacts } = useContactStore();
 
 	const [minutes, setMinutes] = useState(0);
 	const [hint, setHint] = useState<string>();
 
 	const {
-		permissionsGranted: locationPermissionsGranted,
+		permissionGranted: locationPermissionsGranted,
```

Comment:
šta misliš da svu lokacija logiku strpamo u ovaj hook nekako.
Da ti na kraju u ovom ekranu bude samo informacija koja ti treba što je lokacija koja se šalje
`const { location } = useLocation()`